src/NuGet.Client/HttpSource.cs

```diff
@@ -155,7 +156,7 @@ internal class HttpSource
                     }
                 }
 
-                // If the destination file doesn't exist, we can safely perform moving operation.
+                // If the desitnation file doesn't exist, we can safely perform moving operation.
```

Comment:
I remember this typo was fixed by me... Seems I reintroduced it by copy-paste some old version of this file. Fixing.